server/channels/app/content_flagging_setup.go

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package app
```

Comment:
Is this file required?

Reply:
Hmm, I believed I removed both those files, but apparently not. Removed them now.